store/src/java/com/zimbra/cs/service/util/VolumeConfigUtil.java

```diff
@@ -94,6 +100,25 @@ public static void validateCreateVolumeRequest(CreateVolumeRequest request, Volu
             throw VolumeServiceException.INVALID_REQUEST("Volume Name can't be missing, empty or null", VolumeServiceException.BAD_VOLUME_NAME);
         }
 
+        // validate mgbits should be divisible of 8 ( 8,16,24,32 ... )
+        if (volInfoRequest.getMgbits() != 0 && volInfoRequest.getMgbits() % 8 != 0) {
+            throw VolumeServiceException.INVALID_REQUEST("mgbits should be multiple of 8", VolumeServiceException.BAD_VOLUME_MGBITS);
+        }
+
+        // validate mbits should be divisible of 8 ( 8,16,24,32 ... )
+        if (volInfoRequest.getMbits() != 0 && volInfoRequest.getMbits() % 8 != 0) {
```

Comment:
@rupeshtelus I was going through the existing behavior and found that this needs be validated in different way.
```
INSERT INTO volume (id, type, name, path, file_bits, file_group_bits,
    mailbox_bits, mailbox_group_bits, compress_blobs, compression_threshold)
  VALUES (1, 1, 'message1', '/opt/zimbra/store', 12, 8, 12, 8, 0, 4096);
```

Reply:
As we are not sure about its validation lets add validation for `>0` only for these fields.

Reply:
changed & pushed